docs/logs/archiving.md

```diff
@@ -0,0 +1,146 @@
+    <img src="../../images/logs/archiving/ibm-logo.png" style="height:80px;width:90px;margin-top:auto;">
+  </a>
+  <a href="https://azure.microsoft.com/services/storage" target="_blank" title="Azure Cloud Storage">
+    <img src="../../images/logs/archiving/azure-logo.png" style="height:80px;width:104px;margin-top:auto;">
+  </a>
+  <a href="https://cloud.google.com/storage" target="_blank" title="Google Cloud Storage" style="grid-column-start:2">
+    <img src="../../images/logs/archiving/google-logo.png" style="height:80px;width:99px;margin-top:auto;">
+  </a>
+  <a href="https://min.io" target="_blank" title="MinIO Object Storage" style="align-self:center;">
+    <img src="../../images/logs/archiving/minio-logo.png" style="height:30px;width:150px;margin-top:auto;">
+  </a>
+</div>
+
+You can configure a Logs App to forward all the logs it receives
+to an S3-compatible object store.
+
+### How do I configure archiving for my Logs App?
+
+In Sematext web app, go to `Integrations > Apps` view
+and choose `Configure S3` using row context menu (*three-dots* icon)
+of the app whose logs you want to ship to S3:
+
+For Amazon S3, all you have to provide are
+[credentials](#how-to-obtain-credentials-from-aws) and a bucket name:
+
+<img src="../../images/logs/archiving/aws-s3.png" style="height:400px;width:490px">
+
+For any other S3-compatible object store you'll also have to provide
+a service endpoint. Cloud object store providers, like *DigitalOcean
+Spaces* or *IBM Cloud Object Storage* usually refer to it as a 
+`Public Service Endpoint` in bucket configuration. For Minio users
+this would be your Minio public URL.
+
+<img src="../../images/logs/archiving/non-aws-s3.png" style="width:490px;height:477px;">
+
+Paste `Access key ID` and `Secret access key` to the corresponding
+fields.
+
+Enter `Bucket name` (just the simple name, not fully qualified ARN) and
+choose `Compression` (read on for more details about compression) and
+confirm with `Verify and save`.
+
+At this point, Logs App is going to check whether the information is
+valid using the AWS S3 API.
+
+After the check is done you'll see a feedback message confirming
+information validity or an error
+message.
+
+### How to obtain credentials from AWS?
+
+For our Logs App AWS S3 Settings, besides S3 bucket name, you'll need
+`Access Key ID` and `Secret Access Key`.
+
+Log in to your AWS account, go to `IAM > Users` and open (or create) a
+user that you want to use for S3 uploads:
+
+![](attachments/6520901/75759631.png?effects=drop-shadow&height=250)
+
+Click on `Create Access Key`:
+
+![](attachments/6520901/75759633.png?effects=drop-shadow&height=250)
+
+Note down `Access Key ID` and `Secret Access Key` (you can `Download
+Credentials` to a safe place if you like, but it's not necessary).
+
+### Which credentials are required when using AWS S3 Bucket Access Policy?
+
+In order to verify access to your S3 bucket, Logs App will first use
```

Comment:
In order to verify access to your S3 bucket, Logs App will first use

=>

In order to verify access to your S3 bucket, the Logs App will first use